extension/app/scripts.babel/background.js

```diff
@@ -47,6 +53,11 @@ function createResultsHTML(results) {
     const groupClass = 'group ' +
         (groupHasErrors ? 'errors expanded' : 'no-errors collapsed');
 
+    // Skip any tests that didn't run.
+    if (score === '-1') {
```

Comment:
do we want some kind of message indicating tests were not run to make it clear they will be run when pressing the button? Or is the button and label sufficient for this